odoo/tests/__init__.py

```diff
@@ -1,3 +1,10 @@
+"""
+Odoo unit testing framework, based on Python unittest.
+
+Some files as case.py, resut.py, suite.py are higly modified versions of unitest
+See https://github.com/python/cpython/tree/3.10/Lib/unittest for reference files.
+"""
+
 from . import common
 from .common import *
 from . import test_parse_inline_template
```

Comment:
Yes in one of my versions I thought about doing that but since I removed the runner, loader, ... I preferred keep it clear what comes from odoo and what comes from unittest. 

The unittest.mock is really not linked to any other test object. It could be outside of unittest. Since this is imported in many places in odoo I won't change them and adding another way to import it would add confusion I think. I really have a neutral opinion on this point right now.

Reply:
we discussed at work and decided that the current way was good enough